server/mscalendar/views/calendar.go

```diff
@@ -64,3 +64,22 @@ func groupEventsByDate(events []*remote.Event) [][]*remote.Event {
 	}
 	return result
 }
+
+func RenderScheduleItem(s remote.ScheduleItem, timezone string) (string, error) {
+	message := "You have an upcoming event:"
+	start := s.Start.In(timezone).Time()
+	end := s.End.In(timezone).Time()
+
+	message += fmt.Sprintf("\n%s-%s (%s)", start.Format(time.Kitchen), end.Format(time.Kitchen), timezone)
+	if s.Subject == "" {
+		message += fmt.Sprintf("\nNo subject.")
```

Comment:
0/5

```suggestion
		message += fmt.Sprintf("\n(blank)")
```

Reply:
Not sure about this. Any comment @aaronrothschild ?

As a bit of context:
- Since an event can be created without a subject, we have to find a way to show that.
- The normal view (right now) would be the following:
```
You have an upcoming event:
4:00pm-5:00pm (Romance Standard Time)
Subject: Some subject
Location: Some location
```
- My proposal would be to have the subject as "No subject", and therefore...
```
You have an upcoming event:
4:00pm-5:00pm (Romance Standard Time)
No Subject.
Location: Some location
```
- Michael proposal would be to have it as `(blank)`, but not sure if with or without the Subject prefix. Therefore:
```
You have an upcoming event:
4:00pm-5:00pm (Romance Standard Time)
(blank)
Location: Some location
```
or
```
You have an upcoming event:
4:00pm-5:00pm (Romance Standard Time)
Subject: (blank)
Location: Some location
```
- As an extra note, when there is no subject, Microsoft shows `(No subject)` in the calendar.

Reply:
@larkox I would go with "unknown" since the user may assume that they accidentally had a subject of "blank"